projects/android-identity-library.md

```diff
@@ -0,0 +1,148 @@
+# Project Name
+Android Identity Library.
+
+# Project Description
+Libraries and reference applications for working with real-world identity.
+
+The initial focus for this work was mdoc/mDL according to [ISO/IEC 18013-5:2021](https://www.iso.org/standard/69084.html)
+and related standards (mainly ISO 23220 series and ISO 18013-7) on Android. Current focus
+includes other credential formats and operating environments.
+
+## Libraries
+
+The project includes two libraries written in Java and Kotlin. The
+first is `identity` which provides the core building blocks and which
+can also be used on server-side environments. The other is `identity-android`
+which provides Android-specific extensions to the former. It is designed to
+run on Android (API 24 or later) and will take advantage of  Android-specific
+features including hardware-backed Keystore, NFC, Bluetooth
+Low Energy, and so on.
+
+The libraries are intended to be used by Wallet Applications (mobile
```

Comment:
This would need to be extensible to support another use case which is being called Holder Services for People to have a node on the network and a control surface in either a cloud or device based Holder Service or a Hybrid thereof working together e.g. Wallet on Device and their other Holder Service a Personal Data Store (PDS) hosted off device e.g. in the Cloud that is synchronised with their Device Based Wallet/s.

We are considering an OWF open source project for a range of API Adapters to two of our three core API Services for Identity as a Service backed up by their own Personal Data Store.

Reply:
Sure, this is an open source project, we will review requests and welcome contributions. This is liekly not the right place to comment on specific proposals so please do that once the project has been contributed.